src/openam/templates/run-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "openam.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: "{{ template "openam.name" . }}"
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "openam.fullname" . }}-run
+data:
+  run_me.sh: |
+    #!/usr/bin/env bash
+
+    # Instance dir does not exist? Then we need to run setup
+    if [ ! -f ${BASE_DIR}/install.log ] ; then
+      echo "OpenAM config not found. Configuring.."
+      /bin/configure.sh &
+    fi
+    
+    echo "Starting the server"
+    export heap="1g"
+    if [ ! -z "$MAX_HEAP" ] ; then
```

Comment:
MUST: When will this ever be zero, given the line above.  In bash there's a way to set a variable only if it is not defined.